bedrock/settings/base.py

```diff
@@ -950,6 +950,11 @@ def lazy_email_backend():
 FXA_IFRAME_SRC = config('FXA_IFRAME_SRC',
                         default='https://accounts.firefox.com/')
 
+# Bug 1264843: embed alternative FxA servers within partner distributions
+FXA_IFRAME_SRC_DIST_MAP = config('FXA_IFRAME_SRC_DIST_MAP',
```

Comment:
If it's unlikely that other partner repacks will need this, then I'd likely favor simplifying the list to a variable in settings? All we'd need is just one data attribute added to the FxA iframe.